### PR TITLE
Add gofumpt scripts

### PR DESCRIPTION
**Pull request by hakman**
Ref: https://github.com/kubernetes/kops/pull/12865

/cc @olemarkus @johngmyers

### CODE REVIEW

**Comment by hakman** [comment]
@johngmyers Looks like all tests passed.

File: upup/pkg/fi/cloudup/spotinsttasks/elastigroup.go [hunk 1]
```diff
@@ -1774,8 +1772,8 @@ func buildEphemeralDevices(cloud awsup.AWSCloud, machineType *string) ([]*awstas
 }
 
 func buildRootDevice(cloud awsup.AWSCloud, volumeOpts *RootVolumeOpts,
-	imageID *string) (*awstasks.BlockDeviceMapping, error) {
-
+	imageID *string) (*awstasks.BlockDeviceMapping, error,
+) {
```
Comment (johngmyers) [inline upup/pkg/fi/cloudup/spotinsttasks/elastigroup.go:1776]: This formatting is a bit odd.

File: upup/pkg/fi/cloudup/spotinsttasks/elastigroup.go [hunk 2]
```diff
@@ -665,8 +667,7 @@ func (_ *Elastigroup) create(cloud awsup.AWSCloud, a, e, changes *Elastigroup) e
 						group.Compute.LaunchSpecification.LoadBalancersConfig = new(aws.LoadBalancersConfig)
 					}
 
-					group.Compute.LaunchSpecification.LoadBalancersConfig.LoadBalancers =
-						append(group.Compute.LaunchSpecification.LoadBalancersConfig.LoadBalancers, lbs...)
+					group.Compute.LaunchSpecification.LoadBalancersConfig.LoadBalancers = append(group.Compute.LaunchSpecification.LoadBalancersConfig.LoadBalancers, lbs...)
```
Comment (johngmyers) [inline upup/pkg/fi/cloudup/spotinsttasks/elastigroup.go:670]: Slightly less readable, but whatever.

**Review by johngmyers — APPROVED** [review]

**Pull request merged**